P1-Api/Controllers/GameController.cs

```diff
@@ -0,0 +1,113 @@
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using P1_Application;
+using P1_Api.Models.Games;
+using AutoMapper;
+using System.Diagnostics;
+using P1_Application.UseCases;
+using P1_Core.Interfaces;
+using P1_Application.Exceptions;
+using P1_Application.Boundaries;
+using P1_Core.Entities;
+
+namespace P1_Api.Controllers
+{
+    public class GameController : BaseController
+    {
+        private readonly IMediator _mediator;
+        private readonly IMapper _mapper;
+
+        public GameController(ILogger<GameController> logger, IMediator mediator, IMapper mapper, ApplicationContext context) : base(logger, context)
+        {
+            _mediator = mediator;
+            _mapper = mapper;
+        }
+
+        [ProducesResponseType(200)]
+        [ProducesResponseType(500)]
+        [HttpPost("create-game")]
+        public async Task<IActionResult> CreateGame([FromBody] AddOneEntityRequest<Game> request)
+        {
+            try
+            {
+                var response = await _mediator.Send(request);
+                return Ok(response != null ? response : null);
```

Comment:
This is called a ternary. the pattern is (condition ? if true return here : if false return this instead) i.e. `condition ? true : false`

Reply:
Null check is redundant because the statement says -> if null then `null`, if not null then `not null`
Null check can be removed -> `return Ok(response)` does the same thing

Reply:
oh nice catch! I completely overlooked it.